CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Add an additional boolean parameter to `XpringClient`'s constructor which allows toggling between protocol buffer implementations.
```

Comment:
👍 Good job updating the CHANGELOG!

Reply:
Thank you. Really feel like I'm getting good at this whole "software" thing.